src/c++/perf_analyzer/load_manager.cc

```diff
@@ -213,7 +213,7 @@ LoadManager::LoadManager(
       shared_memory_type_(shared_memory_type),
       output_shm_size_(output_shm_size), start_sequence_id_(start_sequence_id),
       sequence_id_range_(sequence_id_range), parser_(parser), factory_(factory),
-      using_json_data_(false), using_shared_memory_(false), next_seq_id_(1)
+      using_json_data_(false), using_shared_memory_(false), curr_seq_id_(0)
```

Comment:
Why switch the concept here from next sequence ID to current sequence ID?

Reply:
The consuming code was always incrementing before using it. If you initialize it at 1, then the actual first sequence ID would be 2. Since what it is actually tracking is the current sequence ID and not the next sequence ID, I renamed it (and fixed the initialization).